src/julia-parser.scm

```diff
@@ -8,7 +8,7 @@
 ;; be an operator.
 (define prec-assignment
   (append! (add-dots '(= += -= *= /= //= |\\=| ^= ÷= %= <<= >>= >>>= |\|=| &=))
-           '(:= => ~ $=)))
+           '(:= => ~ $= ⊻=)))
```

Comment:
I am not sure---I have to admit that my lisp is not up to scratch.  From the code I understand that the `add-dots` operators add a dotted version as well.  `⊻=` already works element wise.  But there is no broadcasting `.⊻` indeed.  Should that be added?